vendor/github.com/go-mysql-org/go-mysql/mysql/util.go

```diff
@@ -401,3 +402,52 @@ func init() {
 		}
 	}
 }
+
+func compareSubVersion(typ, a, b, aFull, bFull string) (int, error) {
```

Comment:
Will replace this with the built-in `go-mysql` functions once we update the `go-mysql` version.